### PR TITLE
Changes to Home page(index) and Downloads page

### PR DESCRIPTION
**Pull request by Devlooper175**
Changes on the Home page(index):

-Changed the "Download Now!" button to "Check out AvdanOS!" as downloading the OS is currently impossible. 

Changes on the Download page:
-Added a note that states, "Download is currently unavailable but you can check out the demo".

### CODE REVIEW

**Comment by froxcey** [comment]
@xAvatar6969 I will handle translation once I can get back to work

File: src/pages/index.tsx [hunk 1]
```diff
@@ -34,9 +34,9 @@ const index = () => {
                 </Button>
               </NextLink>
             )}
-            <NextLink href="/downloads">
+            <NextLink href="/features-beta">
```
Comment (froxcey) [inline src/pages/index.tsx:37]: Maybe change the link to demo?

**Review by froxcey — COMMENTED** [review]
I left some reviews

File: src/pages/index.tsx (hunk 1 above)
Comment (Devlooper175) [reply src/pages/index.tsx:37]: There is already a link for Demo.

Comment (Devlooper175) [reply src/pages/index.tsx:37]: The demo link is under usemobilebar and the features link isn't... so altogether I have just replaced the download with demo.

**Comment by Devlooper175** [comment]
Also, I don't know but for some reason... On the download page under the help section, I am getting [%a]here[%a] instead of link...  Can you please assist?

in the previous version of site where we used:
<TranslatableText injKey="download.tsx/having-trouble">
              Having trouble? Click [%s]here[%s] to get help!
            </TranslatableText>

I am not facing any issues here but on the current one I am facing this... Also attached Snapshot for reference.
image

**Comment by LucasoDevDotTk** [comment]
> Also, I don't know but for some reason... On the download page under the help section, I am getting [%a]here[%a] instead of link... Can you please assist?
> 
> in the previous version of site where we used: Having trouble? Click [%s]here[%s] to get help!
> 
> I am not facing any issues here but on the current one I am facing this... Also attached Snapshot for reference. image

Hi, it seems like you have 3 `div`s, and that's simply not possible.

image
Here is a picture for ya.

**Review by LucasoDevDotTk — CHANGES_REQUESTED** [review]
Please delete 3rd div in line 201, download.tsx.

**Comment by Devlooper175** [comment]
> Please delete the 3rd div in line 201, download.tsx. Sorry, I don't have edit perms, soo I can't fix it for you if you wondered.

That's actually the closing tag for the div on the 95th line although after removing I get this:
image

**Comment by LucasoDevDotTk** [comment]
> > Please delete the 3rd div in line 201, download.tsx. Sorry, I don't have edit perms, soo I can't fix it for you if you wondered.
> 
> That's actually the closing tag for the div on the 95th line although after removing I get this: image

what... wait...

**Comment by LucasoDevDotTk** [comment]
image

**Comment by LucasoDevDotTk** [comment]
while searching it only shows three...

**Comment by LucasoDevDotTk** [comment]
> > Please delete the 3rd div in line 201, download.tsx. Sorry, I don't have edit perms, soo I can't fix it for you if you wondered.
> 
> That's actually the closing tag for the div on the 95th line although after removing I get this: image

Sorry for the confusion, I checked again and line 95 was hidden by github, sorry....

**Comment by Devlooper175** [comment]
> > > Please delete the 3rd div in line 201, download.tsx. Sorry, I don't have edit perms, soo I can't fix it for you if you wondered.
> > 
> > 
> > That's actually the closing tag for the div on the 95th line although after removing I get this: image
> 
> Sorry for the confusion, I checked again and line 95 was hidden by github, sorry....

No problem but I'll try downloading the code again from the repo and comparing...thanks for the help

**Comment by LucasoDevDotTk** [comment]
> 

ok. let me know if you find a solution.

**Comment by Devlooper175** [comment]
Sure I will let you know once i find a solution... Thanks for the help

**Comment by Devlooper175** [comment]
> > 
> 
> ok. let me know if you find a solution.

I wasn't able to find a solution but was able to find a similar issue on demo page (https://avdanos.com/demo) to which Goos replied that there's an issue...

**Comment by LucasoDevDotTk** [comment]
> > > 
> > 
> > 
> > ok. let me know if you find a solution.
> 
> I wasn't able to find a solution but was able to find a similar issue on demo page (https://avdanos.com/demo) to which Goos replied that there's an issue...

hi fixed the broken link to avdanos/demo in PR: fixed link to demo not linking properly #208, but I am not able to find a solution for the problem in download.tsx.

**Comment by Trynera** [comment]
I think I know what the Problem is. I think the <TranslatableText> Tag can't interact with the <a> tag. So I think it could get fixed by doing <TranslatableText link="/demo">Download is not yet available but you can [%a]check out[%a] the</TranslatableText> {' '}. If that doesn't work try: <TranslatableText link="https://avdanos.com/demo">Download is not yet available but you can [%a]check out[%a] the</TranslatableText> {' '}

**Comment by froxcey** [comment]
There's documentation on how to use Translatable**Text** properly in contributing guideline. You cannot use components in TranslatableText because

1. It's hard for translators that doesn't know anything about programming to work with
2. React components are actually some heavily nested JS Objects

Details for the work around is in the contributing guideline. I have to admit it's not the best, but at least it's somewhat simple and works.

**Comment by Trynera** [comment]
> There's documentation on how to use Translatable**Text** properly in contributing guideline. You cannot use components in TranslatableText because
> 
> 1. It's hard for translators that doesn't know anything about programming to work with
> 2. React components are actually some heavily nested JS Objects
> 
> Details for the work around is in the contributing guideline. I have to admit it's not the best, but at least it's somewhat simple and works.

well he didn't use them inside of those but together with them which also doesn't work because well it doesn't get translated if he does that lmao. I think the [%a] was just reminding him about what he needed to do

**Comment by LucasoDevDotTk** [comment]
> > There's documentation on how to use Translatable**Text** properly in contributing guideline. You cannot use components in TranslatableText because
> 
> > 
> 
> > 1. It's hard for translators that doesn't know anything about programming to work with
> 
> > 2. React components are actually some heavily nested JS Objects
> 
> > 
> 
> > Details for the work around is in the contributing guideline. I have to admit it's not the best, but at least it's somewhat simple and works.
> 
> 
> 
> well he didn't use them inside of those but together with them which also doesn't work because well it doesn't get translated if he does that lmao. I think the [%a] was just reminding him about what he needed to do

The thing that's weird is that when I copy his code the link shows for me... so I can't reproduce the bug.

**Comment by Devlooper175** [comment]
> > > > 
> > > 
> > > 
> > > ok. let me know if you find a solution.
> > 
> > 
> > I wasn't able to find a solution but was able to find a similar issue on demo page (https://avdanos.com/demo) to which Goos replied that there's an issue...
> 
> hi fixed the broken link to avdanos/demo in PR: fixed link to demo not linking properly #208, but I am not able to find a solution for the problem in download.tsx.

UPDATE ON THE ISSUE:

I tried fixing this issue but found something strange... I have an old repo and a new repo(one up to date with all commits). There I replace the downloads.tsx of the old repo with the new one and the issue exists... but when I just run the second repo the issue just vanishes. And the surprising part, the code, and the files are exactly the same as in the old repo... I overwrote the old repo to match the commits assuming there might be a file missing... Comparing the folders there was not even a single file different. But the issue doesn't get resolved. Still trying to figure some way out.

**Comment by LucasoDevDotTk** [comment]
> > > > > 
> 
> > > > 
> 
> > > > 
> 
> > > > ok. let me know if you find a solution.
> 
> > > 
> 
> > > 
> 
> > > I wasn't able to find a solution but was able to find a similar issue on demo page (https://avdanos.com/demo) to which Goos replied that there's an issue...
> 
> > 
> 
> > hi fixed the broken link to avdanos/demo in PR: fixed link to demo not linking properly #208, but I am not able to find a solution for the problem in download.tsx.
> 
> 
> 
> UPDATE ON THE ISSUE:
> 
> 
> 
> I tried fixing this issue but found something strange... I have an old repo and a new repo(one up to date with all commits). There I replace the downloads.tsx of the old repo with the new one and the issue exists... but when I just run the second repo the issue just vanishes. And the surprising part, the code, and the files are exactly the same as in the old repo... I overwrote the old repo to match the commits assuming there might be a file missing... Comparing the folders there was not even a single file different. But the issue doesn't get resolved. Still trying to figure some way out.

Maybe something with npm install or outdated libraries?

**Comment by froxcey** [comment]
Heisenbug?

**Comment by LucasoDevDotTk** [comment]
> Heisenbug?

Wat?

**Comment by Devlooper175** [comment]
Thank you everyone for the help. I was able to fix it and have added the commit.

**Comment by LucasoDevDotTk** [comment]
> Thank you everyone for the help. I was able to fix it and have added the commit.

np :)

**Comment by Devlooper175** [comment]
> There's documentation on how to use Translatable**Text** properly in contributing guideline. You cannot use components in TranslatableText because
> 
> 1. It's hard for translators that doesn't know anything about programming to work with
> 2. React components are actually some heavily nested JS Objects
> 
> Details for the work around is in the contributing guideline. I have to admit it's not the best, but at least it's somewhat simple and works.

Thanks, I have updated it to match the guidelines.

File: src/pages/downloads.tsx [hunk 2]
```diff
@@ -96,7 +97,12 @@ const Download = () => {
         <Spacer h={3} />
         <Text className="header" h1>
           <TranslatableText>Give your PC an Upgrade.</TranslatableText>
-        </Text>
+              </Text>
+              <Spacer />
+              <Note style={{ display: 'inline-block' }} type="warning" label="NOTE">
+                  <TranslatableText link="/demo">The download is not available yet, but you can always [%a]check out[%a] the demo.</TranslatableText>
```
Comment (froxcey) [inline src/pages/downloads.tsx:103]: ~~The download is not available yet, but you can always [%a]check out[%a] the demo.~~
The download is not available yet, but you can always check out the [%a]demo[%a].

**Comment by AZProductions** [comment]
Any updates?

**Comment by Devlooper175** [comment]
> Any updates?

I will have to update it to match the updated website home page... I will do it as soon as I get free and about Download page, i guess it is up-to-date.

**Comment by froxcey** [comment]
Just as a reminder, remember to request another review when you're ready.

**Pull request closed without merging**